rd/values.yaml

```diff
@@ -24,6 +24,28 @@ rd-user-profile-api:
       S2S_URL: "{{ tpl .Values.global.s2sUrl $}}"
       S2S_SECRET: AAAAAAAAAAAAAAAA
       IDAM_URL: "{{ .Values.global.idamApiUrl }}"
+      AUTH_IDAM_CLIENT_BASEURL: "https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net"
+
+
+rd-professional-api:
+  java:
+    applicationPort: 8091
+    ingressHost: ""
+    releaseNameOverride: "{{ .Release.Name }}-rd-user-profile-api"
+    #aadIdentityName: rd
```

Comment:
releaseNameOverride should be "{{ .Release.Name }}-rd-professional-api"

Reply:
done